### PR TITLE
Add the linear solve routines (Dgetrs, Dgels) to the lapack64 interface

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: lapack64/lapack64.go [hunk 1]
```diff
@@ -48,6 +48,39 @@ func Potrf(a blas64.Symmetric) (t blas64.Triangular, ok bool) {
...
+// A is singular, and true if this solution was successfully found.
+//
+// The minimization problem solved depends on the input parameters.
+//
+//  1. If m >= n and trans == blas.NoTrans, Dgels finds X such that || A*X - B||_2
+//  is minimized.
+//  2. If m < n and trans == blas.NoTrans, Dgels finds the minimum norm solution of
+//  A * X = B.
+//  3. If m >= n and trans == blas.Trans, Dgels finds the minimum norm solution of
+//  A^T * X = B.
```
Comment (kortschak) [inline lapack64/lapack64.go]: Indent this.

File: cgo/lapack.go [hunk 2]
```diff
@@ -193,6 +200,50 @@ func (impl Implementation) Dgeqrf(m, n int, a []float64, lda int, tau, work []fl
...
 }
 
+// Dgels finds a minimum-norm solution based on the matrices A and B using the
+// QR or LQ factorization. Dgels returns false if the matrix
+// A is singular, and true if this solution was successfully found.
+//
+// The minimization problem solved depends on the input parameters.
+//
+//  1. If m >= n and trans == blas.NoTrans, Dgels finds X such that || A*X - B||_2
+//  is minimized.
```
Comment (kortschak) [inline cgo/lapack.go]: Indentation.

**Comment by kortschak** [comment]
LGTM with minor comments.

File: lapack64/lapack64.go (hunk 1 above)
Comment (btracey) [inline lapack64/lapack64.go]: You mean beyond the numbers? The block itself is indented.

Comment (kortschak) [inline lapack64/lapack64.go]: Yes, make sure the text is at the same level of indentation - otherwise it's harder to read.

**Pull request merged**